frontend/packages/dev-console/src/components/topology/operators/components/OperatorBackedServiceNode.tsx

```diff
@@ -56,7 +50,6 @@ const OperatorBackedServiceNode: React.FC<OperatorBackedServiceNodeProps> = ({
     <g
       ref={refs}
       onClick={onSelect}
-      onContextMenu={editAccess ? onContextMenu : null}
```

Comment:
Just as small question: We do not need a context menu here anymore? :thinking:

Reply:
nope

Reply:
Okay :+1: